### PR TITLE
Support system pybind

### PR DESCRIPTION
**Pull request by blattms**
to ease packaging process.

Currently we will always use the system's pybind. We could make that optional,

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this opm-simulators=3114 please

**Comment by blattms** [comment]
jenkins build this opm-simulators=3114

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=3114 please

**Comment by blattms** [comment]
Converting to draft. Just to scared of Python and need to gain some confidence.

File: CMakeLists.txt [hunk 1]
```diff
@@ -413,7 +416,9 @@ if (OPM_ENABLE_PYTHON)
 
   # -------------------------------------------------------------------------
   # 2: Embed the Python interpreter for keywords like PYACTION and PYINPUT
-  target_include_directories(opmcommon SYSTEM PRIVATE "python/pybind11/include;${PYTHON_INCLUDE_DIRS}")
```
Comment (blattms) [inline CMakeLists.txt:416]: This change is definitely wrong. We need always need the PYTHON_INCLUDE_DIRS

**Comment by blattms** [comment]
jenkins build this opm-simulators=3114 please

File: CMakeLists.txt [hunk 2]
```diff
@@ -210,6 +210,11 @@ if (OPM_ENABLE_PYTHON)
     endif()
     set(PYTHON_INCLUDE_DIRS ${PYTHON_INCLUDE_DIR})
   endif()
+  find_package(pybind11 2.2 CONFIG)
+  if (NOT pybind11_FOUND)
+    # Use full path for reuse with pypi
+    set(pybind11_INCLUDE_DIRS ${CMAKE_SOURCE_DIR}/python/pybind11/include ${PYTHON_INCLUDE_DIRS})
```
Comment (akva2) [inline CMakeLists.txt:216]: please use PROJECT_SOURCE_DIR

Comment (blattms) [reply CMakeLists.txt]: here you are

**Comment by blattms** [comment]
Works on Ubuntu 18.04, Debian, and with socker pypi build.

**Comment by blattms** [comment]
jenkins build this opm-simulators=3114 please

**Comment by joakim-hove** [comment]
Any reason to wait with this?

**Pull request merged**